tools/changelog-generator/missing_entries/3.11.0.0/ai-gateway-license-migration.yml

```diff
@@ -0,0 +1,8 @@
+message: |
+  If the AI Gateway has been enabled for more than a week, upgrades from 3.10 versions to 3.11.0.0 will fail due to a license migration issue. 
```

Comment:
I'm not sure if we want to be more specific or more general in this wording.
If we want to go more specific, I would say `If any AI plugins has been enabled for more than a week` , or rather list the names of AI plugins too. And we might want to exclude Konnect users and only mention on-prem deployment.
If we want to go more generic, I'm fine with the above.

Reply:
> would say If any AI plugins has been enabled for more than a week , or rather list the names of AI plugins too. And we might want to exclude Konnect users and only mention on-prem deployment.

I'm good with more specific, if it's accurate. Then it's more clear which subset of users are affected.

Reply:
Adjusted + added it to the individual plugin changelogs as well.

Reply:
If you want to be more accurate, the Enterprise Edition plugin is more accurate; CE plugin does not affect.

Reply:
I would suggest to not mention CE/EE here, as there may become ambigous on whether it's a CE only plugin or a CE deployment.

Reply:
This is a bit of a grey area, we're avoiding making that distinction right now. Better not to mention at the moment.